src/core/flow.js

```diff
@@ -342,8 +368,20 @@ export function FlowWithPhases({
     return { ...state, G, ctx };
   };
 
+  const onTurnBeginWrap = (state, config) => {
```

Comment:
Let's just inline this inside `startTurn`.

Reply:
ok, i'll do it, for phase and turn